main.go

```diff
@@ -64,17 +64,24 @@ func main() {
 	usage += "\n"
 
 	// Parse arguments
-	arguments, _ := docopt.Parse(usage, nil, true, "taskcluster", true)
+	arguments, _ := docopt.Parse(usage, nil, true, "TaskCluster version 1.0", true)
```

Comment:
`taskcluster version` and `taskcluster --version` should read the version number from the same location, even for a fake version.